<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>selectImage</class>
 <widget class="QWidget" name="selectImage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1175</width>
    <height>749</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Select Ultrasound Image</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget {
	background: rgb(42, 42, 42);
}</string>
  </property>
  <widget class="QLabel" name="selectDataLabel">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>100</y>
     <width>431</width>
     <height>131</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
	font-size: 29px;
	color: rgb(255, 255, 255);
	background-color: rgba(255, 255, 255, 0);
}</string>
   </property>
   <property name="text">
    <string>Select Data and Phantom Files to Generate Ultrasound Image:</string>
   </property>
   <property name="textFormat">
    <enum>Qt::AutoText</enum>
   </property>
   <property name="scaledContents">
    <bool>false</bool>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="phantomPathLabel">
   <property name="geometry">
    <rect>
     <x>790</x>
     <y>290</y>
     <width>271</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
	background-color: rgba(255, 255, 255, 0);
	color: white;
	font-size: 17px;
}</string>
   </property>
   <property name="text">
    <string>Input Path to Phantom file
 (.rf, .rfd, .mat, .bin)</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="textInteractionFlags">
    <set>Qt::NoTextInteraction</set>
   </property>
  </widget>
  <widget class="QPushButton" name="choosePhantomFileButton">
   <property name="geometry">
    <rect>
     <x>792</x>
     <y>390</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	color: white;
	font-size: 16px;
	background: rgb(90, 37, 255);
	border-radius: 15px;
}</string>
   </property>
   <property name="text">
    <string>Choose File</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="phantomPathInput">
   <property name="geometry">
    <rect>
     <x>820</x>
     <y>350</y>
     <width>201</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
	background-color: rgb(249, 249, 249);
	color: black;
}</string>
   </property>
  </widget>
  <widget class="QPushButton" name="clearPhantomPathButton">
   <property name="geometry">
    <rect>
     <x>930</x>
     <y>390</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	color: white;
	font-size: 16px;
	background: rgb(90, 37, 255);
	border-radius: 15px;
}</string>
   </property>
   <property name="text">
    <string>Clear Path</string>
   </property>
  </widget>
  <widget class="QLabel" name="imagePathLabel">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>290</y>
     <width>271</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
	background-color: rgba(255, 255, 255, 0);
	color: white;
	font-size: 17px;
}</string>
   </property>
   <property name="text">
    <string>Input Path to Image file
 (.rf, .rfd, .mat, .bin)</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="textInteractionFlags">
    <set>Qt::NoTextInteraction</set>
   </property>
  </widget>
  <widget class="QPushButton" name="chooseImageFileButton">
   <property name="geometry">
    <rect>
     <x>442</x>
     <y>390</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	color: white;
	font-size: 16px;
	background: rgb(90, 37, 255);
	border-radius: 15px;
}</string>
   </property>
   <property name="text">
    <string>Choose File</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="imagePathInput">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>350</y>
     <width>201</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
	background-color: rgb(249, 249, 249);
	color: black;
}</string>
   </property>
  </widget>
  <widget class="QPushButton" name="clearImagePathButton">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>390</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	color: white;
	font-size: 16px;
	background: rgb(90, 37, 255);
	border-radius: 15px;
}</string>
   </property>
   <property name="text">
    <string>Clear Path</string>
   </property>
  </widget>
  <widget class="QPushButton" name="generateImageButton">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>510</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	color: white;
	font-size: 16px;
	background: rgb(90, 37, 255);
	border-radius: 15px;
}</string>
   </property>
   <property name="text">
    <string>Generate Image</string>
   </property>
  </widget>
  <widget class="QLabel" name="selectImageErrorMsg">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>580</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
	color: rgb(255, 0, 23);
	font-size: 25px;
	background-color: rgba(255, 255, 255, 0);
}</string>
   </property>
   <property name="text">
    <string>Error Msg</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="imagePathFolderLabel">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>290</y>
     <width>271</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
	background-color: rgba(255, 255, 255, 0);
	color: white;
	font-size: 17px;
}</string>
   </property>
   <property name="text">
    <string>Input Path to Image Folder</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="textInteractionFlags">
    <set>Qt::NoTextInteraction</set>
   </property>
  </widget>
  <widget class="QLabel" name="phantomPathFolderLabel">
   <property name="geometry">
    <rect>
     <x>790</x>
     <y>290</y>
     <width>271</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
	background-color: rgba(255, 255, 255, 0);
	color: white;
	font-size: 17px;
}</string>
   </property>
   <property name="text">
    <string>Input Path to Phantom Folder</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="textInteractionFlags">
    <set>Qt::NoTextInteraction</set>
   </property>
  </widget>
  <widget class="QPushButton" name="choosePhantomFolderButton">
   <property name="geometry">
    <rect>
     <x>790</x>
     <y>390</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	color: white;
	font-size: 16px;
	background: rgb(90, 37, 255);
	border-radius: 15px;
}</string>
   </property>
   <property name="text">
    <string>Choose Folder</string>
   </property>
  </widget>
  <widget class="QPushButton" name="chooseImageFolderButton">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>390</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	color: white;
	font-size: 16px;
	background: rgb(90, 37, 255);
	border-radius: 15px;
}</string>
   </property>
   <property name="text">
    <string>Choose Folder</string>
   </property>
  </widget>
  <widget class="QLabel" name="selectImageMethodLabel">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>170</y>
     <width>431</width>
     <height>131</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
	font-size: 29px;
	color: rgb(255, 255, 255);
	background-color: rgba(255, 255, 255, 0);
}</string>
   </property>
   <property name="text">
    <string>Select Method of Image Lookup:</string>
   </property>
   <property name="textFormat">
    <enum>Qt::AutoText</enum>
   </property>
   <property name="scaledContents">
    <bool>false</bool>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="selectIndFilesButton">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>340</y>
     <width>301</width>
     <height>51</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	color: white;
	font-size: 16px;
	background: rgb(90, 37, 255);
	border-radius: 15px;
}</string>
   </property>
   <property name="text">
    <string>Select Individual Files</string>
   </property>
  </widget>
  <widget class="QPushButton" name="selectFoldersButton">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>460</y>
     <width>301</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	color: white;
	font-size: 16px;
	background: rgb(90, 37, 255);
	border-radius: 15px;
}</string>
   </property>
   <property name="text">
    <string>Search Folders for Compatible
Image-Phantom Combinations</string>
   </property>
  </widget>
  <widget class="QWidget" name="sidebar" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>341</width>
     <height>751</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="styleSheet">
    <string notr="true">QWidget {
	background-color: rgb(28, 0, 101);
}</string>
   </property>
   <widget class="QFrame" name="imageSelectionSidebar">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>341</width>
      <height>121</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QFrame {
	background-color: rgb(99, 0, 174);
	border: 1px solid black;
}</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="imageSelectionLabelSidebar">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>0</y>
       <width>191</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel {
	font-size: 21px;
	color: rgb(255, 255, 255);
	background-color: rgba(255, 255, 255, 0);
	border: 0px;
	font-weight: bold;
}</string>
     </property>
     <property name="text">
      <string>Image Selection:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="imageLabel">
     <property name="geometry">
      <rect>
       <x>-60</x>
       <y>40</y>
       <width>191</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel {
	font-size: 16px;
	color: rgb(255, 255, 255);
	background-color: rgba(255, 255, 255, 0);
	border: 0px;
	font-weight: bold;
}</string>
     </property>
     <property name="text">
      <string>Image:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="phantomLabel">
     <property name="geometry">
      <rect>
       <x>-50</x>
       <y>70</y>
       <width>191</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel {
	font-size: 16px;
	color: rgb(255, 255, 255);
	background-color: rgba(255, 255, 255, 0);
	border: 0px;
	font-weight: bold
}</string>
     </property>
     <property name="text">
      <string>Phantom:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="imageFilenameDisplay">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>40</y>
       <width>241</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel {
	font-size: 14px;
	color: rgb(255, 255, 255);
	background-color: rgba(255, 255, 255, 0);
	border: 0px;
}</string>
     </property>
     <property name="text">
      <string>Sample filename </string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="phantomFilenameDisplay">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>70</y>
       <width>241</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel {
	font-size: 14px;
	color: rgb(255, 255, 255);
	background-color: rgba(255, 255, 255, 0);
	border: 0px;
}</string>
     </property>
     <property name="text">
      <string>Sample filename </string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="roiSidebar">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>120</y>
      <width>341</width>
      <height>121</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QFrame {
	background-color: rgb(49, 0, 124);
	border: 1px solid black;
}</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="roiSidebarLabel">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>341</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel {
	font-size: 21px;
	color: rgb(255, 255, 255);
	background-color: rgba(255, 255, 255, 0);
	border: 0px;
	font-weight: bold;
}</string>
     </property>
     <property name="text">
      <string>Region of Interest (ROI) Selection:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="roiTitleLabel">
     <property name="geometry">
      <rect>
       <x>-40</x>
       <y>50</y>
       <width>191</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel {
	font-size: 16px;
	color: rgb(255, 255, 255);
	background-color: rgba(255, 255, 255, 0);
	border: 0px;
	font-weight: bold;
}</string>
     </property>
     <property name="text">
      <string>ROI Name: </string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="roiNameDisplay">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>50</y>
       <width>241</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel {
	font-size: 14px;
	color: rgb(255, 255, 255);
	background-color: rgba(255, 255, 255, 0);
	border: 0px;
}</string>
     </property>
     <property name="text">
      <string>Sample filename </string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="rfAnalysisSidebar">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>360</y>
      <width>341</width>
      <height>121</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QFrame {
	background-color:  rgb(49, 0, 124);
	border: 1px solid black;
}</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="rfAnalysisLabel">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>30</y>
       <width>341</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel {
	font-size: 21px;
	color: rgb(255, 255, 255);
	background-color: rgba(255, 255, 255, 0);
	border: 0px;
	font-weight: bold;
}</string>
     </property>
     <property name="text">
      <string>Radio Frequency Data Analysis</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="exportResultsSidebar">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>480</y>
      <width>341</width>
      <height>121</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QFrame {
	background-color:  rgb(49, 0, 124);
	border: 1px solid black;
}</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="exportResultsLabel">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>301</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel {
	font-size: 21px;
	color: rgb(255, 255, 255);
	background-color: rgba(255, 255, 255, 0);
	border: 0px;
	font-weight: bold;
}</string>
     </property>
     <property name="text">
      <string>Export Results</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QFrame" name="analysisParamsSidebar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>240</y>
     <width>341</width>
     <height>121</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QFrame {
	background-color: rgb(49, 0, 124);
	border: 1px solid black;
}</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="analysisParamsLabel">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>30</y>
      <width>341</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
	font-size: 21px;
	color: rgb(255, 255, 255);
	background-color: rgba(255, 255, 255, 0);
	border: 0px;
	font-weight:bold;
}</string>
    </property>
    <property name="text">
     <string>Analysis Parameter Selection</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
